[c2] operator serialized test coverage summary document

ajyu opened the pull request:
Add a markdown document summarizing the coverage of serialized operator tests. This currently only takes into account what has been covered by the tests with respect to the entire registry of c2 operators.

Next, we will break down the coverage by which operators have unit tests associated with them, which have hypothesis tests, and which have tests more specifically calling assertReferenceChecks.

ezyang commented on caffe2/python/serialized_test/SerializedTestCoverage.md:
```diff
@@ -0,0 +1,725 @@
+# Serialized Test Coverage Report
```
How do you plan to keep this file up to date? Since it is automatically generated, it will be very easy for people to not update it when they increase coverage. One simple way to test for this is to add something to CI which fails if the checked in file is not up-to-date. Another way is to just not check in the generated file.

ajyu replied:
The file is automatically generated whenever someone generates a new serialized test (logic lives in serialized_test_util.py).

ajyu commented:
The failing tests align with what's failing in master.
https://ci.pytorch.org/jenkins/job/pytorch-master/5105/

ajyu committed:
operator serialized test coverage summary document (#13703)

Summary:
Add a markdown document summarizing the coverage of serialized operator tests. This currently only takes into account what has been covered by the tests with respect to the entire registry of c2 operators.

Next, we will break down the coverage by which operators have unit tests associated with them, which have hypothesis tests, and which have tests more specifically calling assertReferenceChecks.
Pull Request resolved: https://github.com/pytorch/pytorch/pull/13703

Reviewed By: dzhulgakov

Differential Revision: D12970810

Pulled By: dzhulgakov

fbshipit-source-id: aab26f018cc8ec44b1fbad604fe3b82e21238021
